CRITICAL - 2021-01-28 09:31:47 --> pg_query(): Query failed: ERROR:  column app_produto_vendas.dara_app does not exist
LINE 1: SELECT &quot;app_produto_vendas&quot;.&quot;id&quot; AS &quot;id&quot;, &quot;app_produto_venda...
                                                  ^
HINT:  Perhaps you meant to reference the column &quot;app_produto_vendas.data_app&quot;.
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', '/Users/vitor/Do...', 196, Array)
#1 /Users/vitor/Documents/GitHub/fetx/system/Database/Postgre/Connection.php(196): pg_query(Resource id #12, 'SELECT "app_pro...')
#2 /Users/vitor/Documents/GitHub/fetx/system/Database/BaseConnection.php(738): CodeIgniter\Database\Postgre\Connection->execute('SELECT "app_pro...')
#3 /Users/vitor/Documents/GitHub/fetx/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT "app_pro...')
#4 /Users/vitor/Documents/GitHub/fetx/system/Database/BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT "app_pro...', Array, false)
#5 /Users/vitor/Documents/GitHub/fetx/app/Models/PadraoModel.php(454): CodeIgniter\Database\BaseBuilder->get()
#6 /Users/vitor/Documents/GitHub/fetx/app/Controllers/Venda.php(133): App\Models\PadraoModel->getQuery(Array)
#7 /Users/vitor/Documents/GitHub/fetx/system/CodeIgniter.php(914): App\Controllers\Venda->cadastro('4284')
#8 /Users/vitor/Documents/GitHub/fetx/system/CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Venda))
#9 /Users/vitor/Documents/GitHub/fetx/system/CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/vitor/Documents/GitHub/fetx/httpdocs/index.php(45): CodeIgniter\CodeIgniter->run()
#11 /Users/vitor/Documents/GitHub/fetx/system/Commands/Server/rewrite.php(34): require_once('/Users/vitor/Do...')
#12 {main}
